dashboard/app/controllers/home_controller.rb

```diff
@@ -115,6 +111,9 @@ def init_homepage
             linkToOverview: script_path(script),
             linkToLesson: script_next_path(script, 'next')
           }
+
+          # Don't include this course in the regular set of recent courses.
+          @recent_courses.reject! {|item| item[:id] == script[:name]}
```

Comment:
afaict `recent_courses_and_scripts` returns a (:name, :description, :link) for each course. where does `item[:id]` come from?